ding/scripts/tests/test_parallel_socket.py

```diff
@@ -0,0 +1,136 @@
+import sys
+import os
+import time
+import logging
+import argparse
+
+from random import random
+from string import ascii_lowercase
+from ding.framework import Parallel
+
+alphabet = [c.encode('ascii') for c in ascii_lowercase]
+
+
+class EasyCounter:
+
+    def __init__(self):
+        self._last = None
+        self._cnt = 0
+
+    def add(self, item):
+        self._last = item
+        self._cnt += 1
+
+    def cnt(self):
+        return self._cnt
+
+    def last(self):
+        return self._last
+
+
+class SockTest:
+
+    @classmethod
+    def main_p0(cls, epoch, interval):
```

Comment:
can you offer a reasonable method name here? `main_p0` is a bit confusing

Reply:
I rename those to receiver, testee, and sender.
In addition, a description is provided above.